### ISSUE TITLE
3.0.0-beta class support is wonky

### ISSUE THREAD

**Issue by othiym23** [feross/standard#59]
```
bauchelain% standard --version
3.0.0-beta
bauchelain% standard
Error: Use JavaScript Standard Style (https://github.com/feross/standard)
  src/models/album-base.js:2:15: Missing space before function parentheses.
  src/models/album-multi.js:8:15: Missing space before function parentheses.
  src/models/album-multi.js:9:4: 'super' is not defined.
  src/models/album-multi.js:16:11: Missing space before function parentheses.
  src/models/album-multi.js:21:10: Missing space before function parentheses.
  src/models/album-multi.js:32:8: Missing space before function parentheses.
  src/models/album-multi.js:46:11: Missing space before function parentheses.
  src/models/album-single.js:4:15: Missing space before function parentheses.
  src/models/album-single.js:5:4: 'super' is not defined.
  src/models/album-single.js:15:11: Missing space before function parentheses.
  src/models/artist.js:2:15: Missing space before function parentheses.
  src/models/artist.js:8:18: Missing space before function parentheses.
  src/models/artist.js:12:11: Missing space before function parentheses.
  src/models/cover.js:4:15: Missing space before function parentheses.
  src/models/cover.js:12:11: Missing space before function parentheses.
  src/models/track.js:4:15: Missing space before function parentheses.
  src/models/track.js:20:11: Missing space before function parentheses.
  src/models/track.js:24:12: Missing space before function parentheses.
  src/models/track.js:32:12: Missing space before function parentheses.
```

[As you can see](https://github.com/othiym23/packard/blob/ce6dda52b7474049473088cce15a49ebd53fb1d3/src/models/album-multi.js), these are pretty standard uses of class syntax, and transpile without issue with `babel`. I haven't had the opportunity to dig in deeper and figure out what espree rules are causing these issues.

**Comment by feross** [comment]
Looks like an eslint bug. I opened an issue: https://github.com/eslint/eslint/issues/1989

**Comment by feross** [comment]
eslint fixed the issue! As of `standard@3.0.0-beta4`, this issue is fixed. :+1:

**Comment by othiym23** [comment]
:tada:
